.pre-commit-config.yaml

```diff
@@ -0,0 +1,15 @@
+exclude: (.*/vendor/.*)
```

Comment:
This excludes any file in a `vendor` directory. I assume those are copied from elsewhere and should probably not be touched by us.